### DISCUSSION TITLE
First attempt to use the Power of the Community to Fix a Problem ;-)

### DISCUSSION THREAD

**Discussion by LikwidShadow** [nds-swe/swdt#22] [Q&A]
Hello Everybody

If i cURL the Link [https://nds-swe.github.io/swdt/](url) i see there is a bug in the HTML Code. It is clear to me i can make an Issue and Dominik Fix it ;-) but i think the right Workflow is to Fork the Repository, fix the bug itself and create a Pull request.
So, i have fork the repository but i have no Idea where i can find the HTML Code in this repository.
Maybe someone in the Community can help me??

THX Simon

**Comment by mambax** [comment]
Hi @LikwidShadow 

Nice discussion 🎉

Interesting, maybe I made the mistake in src/pages/index.js. The website is rendered from code with docusaurus, there is no "html" html except in the build result 🤓
If I did not make the bug myself you might have spotted something in docusaurus 🦖

Anyway, nice that you found the way 🤩

PS: Your proposal about how to propose a fix with a PR is correct. Note that for very small changes GitHub supports that you edit the file directly online (GH will make a fork & PR under the hood for you) 🍺
